test/e2e/page-objects/pages/send/send-token-page.ts

```diff
@@ -196,6 +199,25 @@ class SendTokenPage {
       text: address,
     });
   }
+
+  /**
+   * Verifies that a specific warning message is displayed on the send token screen.
+   *
+   * @param warningText - The expected warning text to validate against.
+   * @returns A promise that resolves if the warning message matches the expected text.
+   * @throws Assertion error if the warning message does not match the expected text.
+   */
+  async check_warningMessage(warningText: string): Promise<void> {
+    console.log(`Checking if warning message "${warningText}" is displayed`);
+    const warning = await this.driver.findElement(this.warning);
+    const text = await warning.getText();
+    assert.equal(
+      text,
+      warningText,
+      `Expected warning message to be "${warningText}", got "${text}"`,
+    );
```

Comment:
to avoid flakiness, it's better to use `waitForSelector` to wait for a element with a locator that contains the expected text.
```
await this.driver.waitForSelector({
        css: this.warning
        text: warningText,
      });
```